### ISSUE TITLE
Evaluate the quality-monitor github action

### ISSUE THREAD

**Issue by laeubi** [eclipse-platform/eclipse.platform.releng.aggregator#1733]
See:
- https://github.com/uhafner/quality-monitor

This is similar to what Jenkins already offers but gives a much more integrated/nicer feedback and might even replace the Junit result panel we currently have.

**Comment by akurtakov** [comment]
It's worth trying, maybe in PDE first?

**Comment by jukzi** [comment]
please not. we have enough trouble with the jenkins plugins.

**Comment by merks** [comment]
@jukzi

I don't think this is nice commentary.  The title here is to evaluate something.  Surely its reasonable to evaluate something. 

I see both you and @laeubi doing so very much work to make things better.   Sure, sometimes there are growing pains, sometimes there are differences of opinion, and sometimes some things appear worse than before.  For example, the warning elimination effort is great, but already there have been two regressions as a result, one of which cost me the better part of a day to fix.   But I don't say, "eliminate warnings, please don't!", because I respect that you are doing very good things in general with the best of intentions.  Of course none of us is perfect, so mistakes can and do happen to the best of us.  

I think it's important to remain respectful and to recognize the good work that all the other people are doing.  It's not as if @laeubi  and I always see eye-to-eye, but I greatly respect his huge contribution to releng improvements, often with complaints as his only thank you.  It would be good to be just a little nicer.

**Comment by laeubi** [comment]
Yes I'll probably pick that up later.
